testdata/project-v3-multigroup/controllers/apps/suite_test.go

```diff
@@ -47,30 +47,28 @@ func TestAPIs(t *testing.T) {
 		[]Reporter{printer.NewlineReporter{}})
 }
 
```

Comment:
See: https://github.com/kubernetes-sigs/kubebuilder/pull/1789
/hold 
until the above one gets merged it rebased.

Reply:
Ping me for a `lgtm`label after a rebase